app.rb

```diff
@@ -0,0 +1,142 @@
+require_relative 'person'
+require_relative 'classroom'
+require_relative 'book'
+require_relative 'rental'
+require_relative 'student'
+require_relative 'teacher'
+
+def list_all_books(books)
+  puts 'List of books:'
+  puts
+  books.each_with_index do |book, index|
+    puts "#{index + 1}. Title: #{book.title}, Author: #{book.author}"
+    puts
+  end
+end
+
+def list_all_people(people)
+  puts 'List of people:'
+  puts
+  people.each_with_index do |person, index|
+    puts "#{index + 1}. Name: #{person.name}, ID: #{person.id}"
+    puts
+  end
+end
+
+def create_person(people)
+  puts 'Select person type:'
+  puts '1. Student'
+  puts '2. Teacher'
+  print 'Choice: '
+  type = gets.chomp.to_i
+
+  case type
+  when 1
+    create_student(people)
+  when 2
+    create_teacher(people)
+  else
+    puts 'Invalid option!'
+  end
```

Comment:
- [ ] When I try to create either a student or teacher, I get this error below. Please fix it.

<img width="1144" alt="Screenshot 2023-06-09 at 10 56 26" src="https://github.com/MMhaidari/school_library/assets/64774936/44529ac8-65f1-4740-8ce3-3d004925ce28">